### PR TITLE
Update PDex to 2.0.0-ballot

### PR DESCRIPTION
**Pull request by sfc42**
Signed-off-by: Susan Crowell <sfc@us.ibm.com>

### CODE REVIEW

File: fhir-examples/src/main/resources/json/profiles/fhir-ig-davinci-pdex/Bundle-1000000-1.json [hunk 1]
```diff
@@ -0,0 +1 @@
...3 PM</p><h3>Agents</h3><table class=\"grid\"><tr><td>-</td><td><b>Type</b></td><td><b>Role</b></td><td><b>Who</b></td></tr><tr><td>*</td><td>Transmitter <span style=\"background: LightGoldenRodYellow; margin: 4px; border: 1px solid khaki\"> (<a href=\"http://hl7.org/fhir/us/core/STU3.1/CodeSystem-us-core-provenance-participant-type.html\">US Core Provenance Participant Type Extension Codes</a>#transmitter)</span></td><td>Transcriber <span style=\"background: LightGoldenRodYellow; margin: 4px; border: 1px solid khaki\"> (<a href=\"http://terminology.hl7.org/3.0.0/CodeSystem-contractsignertypecodes.html\">Contract Signer Type Codes</a>#TRANS)</span></td><td><a href=\"Organization-Payer1.html\">Organization/Payer1</a> \"Payer 1\"</td></tr></table></div>"},"target":[{"reference":"Bundle/1000000-1"}],"recorded":"2019-12-31T20:26:23.217+00:00","agent":[{"type":{"coding":[{"system":"http://hl7.org/fhir/us/core/CodeSystem/us-core-provenance-participant-type","code":"transmitter","display":"Transmitter"}]},"role":[{"coding":[{"system":"http://terminology.hl7.org/CodeSystem/contractsignertypecodes","code":"TRANS"}]}],"who":{"reference":"Organization/Payer1"}}]},"response":{"status":"200"}}]}
```
Comment (lmsurpre) [inline fhir-examples/src/main/resources/json/profiles/fhir-ig-davinci-pdex/Bundle-1000000-1.json:1]: we've started to move spec examples out of fhir-examples and into the `src/test/resources` of the `fhir-ig-` project that wraps the actual IG

File: conformance/fhir-ig-davinci-pdex/src/main/resources/hl7/fhir/us/davinci-pdex/package/.index.json [hunk 2]
```diff
@@ -6,21 +6,21 @@
       "resourceType": "ImplementationGuide",
       "id": "hl7.fhir.us.davinci-pdex",
       "url": "http://hl7.org/fhir/us/davinci-pdex/ImplementationGuide/hl7.fhir.us.davinci-pdex",
-      "version": "1.0.0"
+      "version": "2.0.0-ballot"
```
Comment (lmsurpre) [inline conformance/fhir-ig-davinci-pdex/src/main/resources/hl7/fhir/us/davinci-pdex/package/.index.json:9]: I think we want to keep 1.0.0 around and add 2.0.0 as a new package
1. move the 1.0.0 stuff from hl7/fhir/us/davinci-pdex/ to hl7/fhir/us/davinci-pdex/100
2. add the new 2.0.0-ballot stuff in a new package hl7/fhir/us/davinci-pdex/200

Comment (sfc42) [reply conformance/fhir-ig-davinci-pdex/src/main/resources/hl7/fhir/us/davinci-pdex/package/.index.json:9]: OK, I think I'll start over with a new branch so it's easier to review how the current files went to hl7/fhir/us/davinci-pdex/100.  Closing this pull request and deleting the branch.

File: fhir-examples/src/main/resources/json/profiles/fhir-ig-davinci-pdex/Bundle-1000000-1.json (hunk 1 above)
Comment (sfc42) [reply fhir-examples/src/main/resources/json/profiles/fhir-ig-davinci-pdex/Bundle-1000000-1.json:1]: Done in https://github.com/IBM/FHIR/pull/3424.

File: conformance/fhir-ig-davinci-pdex/src/main/resources/hl7/fhir/us/davinci-pdex/package/.index.json (hunk 2 above)
Comment (sfc42) [reply conformance/fhir-ig-davinci-pdex/src/main/resources/hl7/fhir/us/davinci-pdex/package/.index.json:9]: Done in https://github.com/IBM/FHIR/pull/3424.

**Pull request closed without merging**